### PR TITLE
Wait in test case till the runs are recorded.

### PR DESCRIPTION
**Pull request by sagarkapare**
Build is green here - http://builds.cask.co/browse/CDAP-DUT3789-1

### CODE REVIEW

**Comment by sagarkapare** [comment]
@bdmogal Please take a look. Thanks!

File: cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/handlers/WorkflowHttpHandlerTest.java [hunk 1]
```diff
@@ -460,6 +460,7 @@ public void testMultipleWorkflowInstances() throws Exception {
   }
 
   private void verifyMultipleConcurrentRuns(Id.Program workflowId) throws Exception {
+    verifyProgramRuns(workflowId, ProgramRunStatus.RUNNING.name(), 1);
```
Comment (bdmogal) [inline cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/handlers/WorkflowHttpHandlerTest.java]: shouldn't we wait for 2 runs?

Comment (sagarkapare) [inline cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/handlers/WorkflowHttpHandlerTest.java]: verifyProgramRuns actually wait for 2 runs if you check the implementation. However I agree contract of this method is not clear. Will fix in separate PR.

Comment (bdmogal) [inline cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/handlers/WorkflowHttpHandlerTest.java]: ok :) Seems like ``verifyProgramRuns`` is actually ``verifyProgramRunsMoreThan``

**Comment by bdmogal** [comment]
LGTM

**Pull request merged**